velox/functions/prestosql/tests/SliceTest.cpp

```diff
@@ -13,117 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "velox/common/base/tests/GTestUtils.h"
-#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
-
-using namespace facebook::velox;
-using namespace facebook::velox::test;
-using namespace facebook::velox::functions::test;
+#include "velox/functions/lib/Slice.h"
+#include "velox/functions/lib/tests/SliceTestBase.h"
 
+namespace facebook::velox::functions::prestosql {
 namespace {
 
-class SliceTest : public FunctionBaseTest {
+class SliceTest : public SliceTestBase {
  protected:
   static const vector_size_t kVectorSize{1000};
-
-  void testSlice(
-      const std::string& expression,
-      const std::vector<VectorPtr>& parameters,
-      const ArrayVectorPtr& expectedArrayVector) {
-    auto result = evaluate<ArrayVector>(expression, makeRowVector(parameters));
-    assertEqualVectors(expectedArrayVector, result);
-    EXPECT_NO_THROW(expectedArrayVector->checkRanges());
+  static void SetUpTestCase() {
+    SliceTestBase::SetUpTestCase();
+    registerBigintSliceFunction("");
```

Comment:
Shall we register Presto functions to make sure the bigint slice function is included in them?